src/test/shared/filesystemUtilities.test.ts

```diff
@@ -59,6 +59,7 @@ describe('filesystemUtilities', () => {
             const nestedTempDirPath = await makeTemporaryToolkitFolder('nestedSubfolder', 'moreNestedSubfolder')
 
             foldersToCleanUp.push(nestedTempDirPath)
+            foldersToCleanUp.push(path.join(tempDirPath, 'nestedSubfolder'))
```

Comment:
What's the difference between `nestedTempDirPath` and `path.join(tempDirPath, 'nestedSubfolder')`?

Reply:
`nestedTempDirPath` appears to be the child folder of the latter.

Reply:
nestedTempDirPath is `...\nestedSubfolder\moreNestedSubfolder`
now explicitly deleting `...\nestedSubfolder\` since nestedTempDirPath was the path getting deleted